README.md

```diff
@@ -19,159 +19,20 @@ Arctic has been under active development at [Man Group](https://www.man.com/) si
 
 ---
 
-## Man Group is currently building the next generation of Arctic.
+## :mega: ArcticDB coming soon! :mega:
 
-This will offer the same intuitive Python-centric API whilst utilizing a custom C++ storage engine and modern S3 compatible object storage to provide a timeseries database that is:
+[ArcticDB](https://www.man.com/man-group-brings-powerful-dataframe-database-product-arcticdb-to-market-with-bloomberg), Man Group's high-performance Python-native DataFrame database will soon be available on GitHub. ArcticDB is the next generation of Arctic and is designed to be a drop-in replacement for Arctic, providing a platform that is:
 
 * **Fast**: Capable of processing billions of rows in seconds
 * **Flexible**: Designed to handle complex real-world datasets
 * **Familiar**: Built for the modern Python Data Science ecosystem - Pandas In/Pandas Out!
 
-For more information, please contact us at [arctic@man.com](arctic@man.com). 
+This repository **does not contain the code for ArcticDB**, but instead contains the first version of the Arctic platform. Please note that this repository is now in maintenance mode and all ongoing development efforts will be focused on ArcticDB.
```

Comment:
Matt - this line is slightly confusing.  Would it be clearer is you said it "contains the older (first) version of the Arctic platform".